app/controllers/public_pages_controller.rb

```diff
@@ -38,4 +38,6 @@ def tax_questions; end
   def stimulus_recommendation; end
 
   def sms_terms; end
+
+  def diy; end
```

Comment:
FYI - this `def diy; end` isn't strictly necessary (at least, from what I remember from looking into this a few months ago). No change required, just wanted to share that, and also if you have learned it _is_ required, then I want to know that.

Reply:
oh, i didn't know that! im going to leave it because seems like the pattern in this controller is to create an empty method to signify a custom controller action that renders a corresponding template, but i guess we could decide to remove all of those if we want to.